components/LocalList.php

```diff
@@ -168,7 +168,14 @@ protected function getOrderType()
 
     protected function getOrderTypes()
     {
-        return Location::current()->getOrderTypeOptions();
+        $types = [];
+        foreach (Locations_model::isEnabled()->get() as $location) {
+            foreach ($location->getOrderTypeOptions() as $locationOrderType) {
+                if (!in_array($locationOrderType, $types))
+                    $types[] = $locationOrderType;
+            }
+        }
+        return $types;
```

Comment:
Not sure this is necessary as `Location::current()->getOrderTypeOptions()` returns all the registered order types and not just the location's order types..

Reply:
Oops, I should start reading comments before reviewing!

We can use `OrderTypes::instance()->listOrderTypes()` to list all registered order types..

Reply:
Location::current() isnt always present... in that case maybe its use Locations_model::first()->getOrderTypeOptions() ?